dashboard/app/models/oauth_credentials.rb

```diff
@@ -0,0 +1,28 @@
+
+#
+# =========================================================================
+#   LTI uses Oauth 1 for SSO, and generates a HMAC-SHA1 signature
+#
+#   Stub model for now until it is decided how credentials should be
+#   administered. This one is used in the test suite.
+#
+#   To create consumer_key/consumer_secret
+#   > SecureRandom.uuid.gsub("-", "")
+# =========================================================================
+#
+
+class OauthCredentials
+  @@credentials = {}
+  @@credentials["f10ee9fc082219227976f2c1603a3d77"] = {
```

Comment:
In the final implementation, I assume these secrets would be chef secrets in the cdo config?